jwst/assign_wcs/niriss.py

```diff
@@ -133,8 +133,8 @@ def imaging(input_model, reference_files):
 
 def imaging_distortion(input_model, reference_files):
     distortion = AsdfFile.open(reference_files['distortion']).tree['model']
-    # Convert to arcsec
-    transform = distortion | Scale(1/60) & Scale(1/60)
+    # Convert to deg.  Output of distortion model is in arcsec.
+    transform = distortion | Scale(1/3600) & Scale(1/3600)
```

Comment:
Eventually (after all instruments have correct reference file) the conversion to degrees should be moved to the V2V3 to sky transform if possible.